client/public/locales/en/translation.json

```diff
@@ -507,7 +507,8 @@
     "success": {
       "saveWhat": "{{type}} {{what}} was successfully saved.",
       "save": "{{type}} was successfully saved.",
-      "applicationDeleted": "{{appIDCount}} application(s) successfully deleted.",
+      "applicationDeleted": "{{appName}} application(s) successfully deleted.",
```

Comment:
I'm not a native speaker but I would suggest `Application {{appName}} successfully deleted.`